src/test.sig

```diff
@@ -1,7 +1,10 @@
 signature TEST =
 sig
   exception Failed
-
+  val testsRan : int ref
+  val testsPassed : int ref
   val test : (unit -> bool) -> bool
-  val assert : bool -> bool
```

Comment:
I propose leaving an `assert : bool -> bool` and adding the `assertEq` function.